misc/vtr-libs/meta.yaml

```diff
@@ -0,0 +1,49 @@
+# Use `conda-build-prepare` before building for a better version string.
```

Comment:
shouldn't this go alongside the other `vtr` packages in https://github.com/hdl/conda-eda/tree/master/pnr?